cmds/core/sluinit/uinit.go

```diff
@@ -52,15 +52,15 @@ func main() {
 	tpmDev, err := tpm.GetHandle()
 	if err != nil {
 		log.Printf("tpm.getHandle failed. err=%v", err)
-		os.Exit(1)
+		unmountAndExit()
```

Comment:
you may want to consider registering unmountAndExist as a defer and then you can just call return in all these locations